docs/release-notes.txt

```diff
@@ -6,6 +6,49 @@ Release Notes
 Version 0.12
 ------------
 
+Version 0.12.3
+^^^^^^^^^^^^^^
+
+**Release Date:** April 27, 2020
+
+- Add a tutorial for the new (experimental) Native API.
+
+- Add support for locally testing experiments via ``det e create --local``.
+
+- Add :class:`determined.experimental.Determined` class for accessing Experiment, Trial, and Checkpoint objects.
+
+- TensorBoard logs now appear under the ``storage_path`` for ``shared_fs`` checkpoint configurations.
+
+- Allow commands, notebooks, shells, and TensorBoards to be killed before they are scheduled.
+
+- Print container exit reason in trial logs.
+
+- Choose a better default for the ``--tail`` option of command logs.
+
+- Add additional REST API endpoints for trials.
+
+- Support a startup script inside agent docker images.
+
+- Master and agent Docker containers will have the 'unless-stopped' restart policy by default when using ``det-deploy local``.
+
+- Prevent the ``det trial logs -f`` command from waiting for too long after the trial being watched reaches a terminal state.
+
+- Fix bug where logs disappear when an image is pulled.
+
+- Fix bug that affected the use of :class:`LRScheduler <determined.pytorch.LRScheduler>` in :class:`PyTorchTrial <determined.pytorch.PyTorchTrial>` for multi-GPU training.
+
+- Fix bug after master restart where some errored experiments would show progress indicators.
+
+- Fix bug that stops loading user script if user script exits the system.
```

Comment:
I had a hard time with this one, it doesn't read correctly but I'm not sure what it's trying to say. Could you get some clarification here?

Reply:
:heavy_plus_sign: Unclear what the bug (or fix) actually is?

Reply:
"this one" being "loading user script"

Reply:
let's just remove this one, I'm pretty sure it has only affected our SEs during dogfooding.